src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  gap: $cellGap;
+  flex-wrap: wrap;
+  width: calendarWidth($cellSize, $weekLength, $calendarPadding, $cellGap);
+  padding: $calendarPadding;
+
+  &__cell {
+    display: flex;
+    width: $cellSize;
+    height: $cellSize;
+    background-color: $cellColor;
+    border: 1px solid #000;
+    cursor: pointer;
+    transition-duration: 0.5s;
```

Comment:
it's always better to set `transition-property` to specify what exactly you'd like to change
Also use transition `shorthand`